src/_includes/index_cta.html

```diff
@@ -22,15 +22,6 @@ <h2 class="modal-title">
           <li>Pairing sessions to help you draft and evolve your Tiltfile.</li>
           <li>Help from our content strategists on how to talk about and teach Tilt.</li>
         </ul>
-        <hr/>
-        <p>
-          Not sure if a formal program is right for you?
```

Comment:
maybe keep this part, but direct them to the Slack channel?

Reply:
Done!

<img width="848" alt="Screen Shot 2021-08-24 at 5 13 57 PM" src="https://user-images.githubusercontent.com/841263/130690940-cf41fe7c-897a-4198-89cb-f2e678d0f3bc.png">